pull_request_template.md

```diff
@@ -1,11 +1,11 @@
 ### What this PR does (required):
-- list PR functionality here
+- created model for profile schema with requirements and added optional properties based on what it looks like the profile model will need in the future
 
```

Comment:
Just so you are aware, the `pull_request_template.md` will be what is prefilled in the comment when you create a new PR on GitHub. You don't need to edit this file directly for every pull request.